app/src/pages/clinic.tsx

```diff
@@ -0,0 +1,118 @@
+import clinics from '@public/data/clinics.json'
+import type { GetServerSideProps, NextPage } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { ChangeEvent, FormEvent, useState } from 'react'
+import Image from 'next/image'
+
+import ButtonLink from '@components/ButtonLink'
+
+const Clinic: NextPage = () => {
+  const { t } = useTranslation('common')
+  const [filteredClinics, setFilteredClinics] = useState<typeof clinics>([])
+  const [selectedClinic, setSelectedClinic] = useState<
+    typeof clinics[0] | undefined
+  >(undefined)
+  const [search, setSearch] = useState('')
+
+  const handleSearch = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setSelectedClinic(undefined)
+    const filtered = clinics.filter((clinic) => clinic.zip === search)
```

Comment:
Does this only return a clinic if you've selected that exact zip code? Do we want to widen that out at all? If so, how? @danielladevera-nava This might be an additional ticket (not for this PR).

Reply:
@rocketnova I checked in with Daniella about this in this slack thread: https://nava.slack.com/archives/C03CGSMQ0JV/p1658770251910699

Reply:
I think your slack thread doesn't address the question of exact zip code match.

Reply:
If we want this, we should pull this out into a separate ticket.